app/src/main/java/fr/neamar/kiss/loader/LoadContactsPojos.java

```diff
@@ -53,8 +54,24 @@ public LoadContactsPojos(Context context) {
                             .getColumnIndex(ContactsContract.Contacts.LOOKUP_KEY));
                     contact.timesContacted = Integer.parseInt(cur.getString(cur
                             .getColumnIndex(ContactsContract.CommonDataKinds.Phone.TIMES_CONTACTED)));
-                    contact.setName(cur.getString(cur
-                            .getColumnIndex(ContactsContract.CommonDataKinds.Phone.DISPLAY_NAME)));
+                    contact.setName(cur.getString(cur.getColumnIndex(ContactsContract.CommonDataKinds.Phone.DISPLAY_NAME)));
+
+                    String contactID = cur.getString(cur.getColumnIndex(ContactsContract.CommonDataKinds.Phone.CONTACT_ID));
+
+                    Cursor cur2 = context.getContentResolver().query(
```

Comment:
Ok, I was able to do it using a raw query on the database from command line. I believe it's not possible to JOIN tables the android way, but, if I don't remember wrong, the android API does allow the execution of raw queries, and that would solve the problem.  
The only downside is that it may be considered a less clean implementation.